specification/examples/responses/GET_Consent/errors/invalid-status-parameter.yaml

```diff
@@ -1,6 +1,6 @@
 ConsentInvalidStatusParameterError:
   summary: status parameter invalid
-  description: 422 error response bundle for an invalid status parameter
+  description: 400 error response bundle for an invalid status parameter
```

Comment:
Is this a team decision to change the status codes to 400? As at the time it was debated 400 or 422, with 422 being considered the right status choice at the time.

> A 422 status code indicates that the server was unable to process the request because it contains invalid data